integration-tests/contracts/ethereum_vrfv2plus_contracts.go

```diff
@@ -1212,3 +1163,64 @@ func (v *EthereumVRFV2PlusWrapperLoadTestConsumer) GetLoadTestMetrics(ctx contex
 		FastestResponseTimeInSeconds:         nil,
 	}, nil
 }
+
+func extractVRFCoordinatorV25SubscriptionCanceledEvent(events []seth.DecodedTransactionLog) (*vrf_coordinator_v2_5.VRFCoordinatorV25SubscriptionCanceled, error) {
+	var cancelEvent vrf_coordinator_v2_5.VRFCoordinatorV25SubscriptionCanceled
+	for i, event := range events {
+		if len(event.Topics) == 0 {
+			return nil, fmt.Errorf("no topics in event %d", i)
+		}
+		switch event.Topics[0] {
+		case vrf_coordinator_v2_5.VRFCoordinatorV25SubscriptionCanceled{}.Topic().String():
```

Comment:
why we cant use geth wrapper's method for parsing log , the same way I did [here](https://github.com/smartcontractkit/chainlink/blob/a4b0cca782793f4f49979f7bdb6dc396ae78bc62/integration-tests/contracts/ethereum_vrfv2plus_contracts.go#L214)


this applies for all events. in this case wrapper should have `ParseSubscriptionCanceled`

Reply:
@Tofel could you answer that? It makes sense to me. But, we would need to have `types.Log` instead of `seth.DecodedTransactionLog`

Reply:
guys, feel free to experiment, there's probably a way to get native `types.Log` from transaction or receipt. If you really need it, I am sure you will find a way to find them. If not, let's stick to the current implementation.

Reply:
@iljapavlovs I talked with Bartek and he confirmed that we do not have `types.Log` in the seth transaction details. So, we can't use `Parse` function.

Reply:
@iljapavlovs there is a way. See example below and let me know if you prefer it over the existing solution:
```
func (v *EthereumVRFCoordinatorV2_5) CancelSubscription(subID *big.Int, to common.Address) (*seth.DecodedTransaction, *vrf_coordinator_v2_5.VRFCoordinatorV25SubscriptionCanceled, error) {
	tx, err := v.client.Decode(v.coordinator.CancelSubscription(
		v.client.NewTXOpts(),
		subID,
		to,
	))
	if err != nil {
		return nil, nil, err
	}
	var cancelEvent *vrf_coordinator_v2_5.VRFCoordinatorV25SubscriptionCanceled
	for _, log := range tx.Receipt.Logs {
		for _, topic := range log.Topics {
			if topic.Cmp(vrf_coordinator_v2_5.VRFCoordinatorV25SubscriptionCanceled{}.Topic()) == 0 {
				cancelEvent, err = v.coordinator.ParseSubscriptionCanceled(*log)
				if err != nil {
					return nil, nil, fmt.Errorf("parsing SubscriptionCanceled log failed, err: %w", err)
				}
			}
		}
	}
	return tx, cancelEvent, err
}
```

Reply:
yes, I prefer this way much more!

Reply:
@iljapavlovs fixed in ab04fe41349ebfd0bc799bedac1648c2fa5f0151